data/uhc_pack/functions/pre_generation/stage/step/marker.mcfunction

```diff
@@ -0,0 +1,5 @@
+execute if score Dimension uhcPG matches -1 in minecraft:the_nether run function uhc_pack:pre_generation/stage/step/marker/create
+execute if score Dimension uhcPG matches 0 in minecraft:overworld run function uhc_pack:pre_generation/stage/step/marker/create
+execute if score Dimension uhcPG matches 1 in minecraft:the_end run function uhc_pack:pre_generation/stage/step/marker/create
+
+execute as @e[type=minecraft:armor_stand,tag=pre_generator,tag=new,limit=1] run function uhc_pack:pre_generation/stage/step/marker/relocate
```

Comment:
In terms of the organisation of the functions, my preference is to avoid the deeply nested directory structures. Some of these functions aren't really hierarchies. For example, this could be something like `uhc_pack:pre_generation/marker/relocate` instead. I think though it would be easiest for me to to a pass over the directory structure after merging this.